src/components/Login/Form.tsx

```diff
@@ -72,7 +72,7 @@ export function Login(props: InjectedProps) {
             </div>
 
             <div className="form__footer">
-              <button type="reset" className="form__button--reset" name="reset" value="Reset">
+              <button type="reset" className="button button__reset" name="reset" value="Reset">
```

Comment:
How about

```suggestion
              <button type="reset" className="button--reset" name="reset" value="Reset">
```

Maybe we should not use the class `button` everywhere, just `button--filter` or `button--reset` 👀